### DISCUSSION TITLE
feature: integration with Jenkinsfile

### DISCUSSION THREAD

**Discussion by idelice** [LazyVim/LazyVim#3474] [Extra Requests]
### Did you check the docs?

- [X] I have read all the LazyVim docs

### Is your feature request related to a problem? Please describe.

I have no LSP attached to a Jenkinsfile, nor treesitter or cmp. 

Neovim sees it as normal text.

### Describe the solution you'd like

I would like an integration that enables treesitter parsing, cmp and linter for Jenkinfiles

### Describe alternatives you've considered

n/a

**Comment by mikavilpas** [comment]
Looks like a new parser for groovy was merged around two (2) months ago https://github.com/nvim-treesitter/nvim-treesitter/pull/5957

To activate it, maybe adding it to `ensure_installed` would be enough.

(this will only address the treesitter parser part of the request)
